packages/augur-ui/src/modules/common/selection.tsx

```diff
@@ -160,7 +160,7 @@ class Dropdown extends React.Component<DropdownProps, DropdownState> {
           [Styles.openTop]: openTop,
           [`${activeClassName}`]: showList,
           [Styles.showColor]: showColor,
-          [`${Styles[`showColor-${selected.value + 1}`]}`]: selected && showColor
+          [`${Styles[`showColor-${selected ? selected.value + 1 : 1}`]}`]: selected && showColor
```

Comment:
should we get rid of the selected check after the colon

Reply:
selected can be null. or an object or string or number. a bit crazy! let's leave it for now.